### PR TITLE
Pt 163423623 rename mnesia db -- Clarify DB path config

### PR DESCRIPTION
**Pull request by lucafavatella**

### CODE REVIEW

**Comment by lucafavatella** [comment]
I created node with `make prod-build`. I created config with `printf '%s\nchain:\n  persist: true\npeers: []\n' '---' > aeternity.yaml` - i.e. I kept db_path unset. Attempt to start node with old dB generated - among others - log `Database cannot be loaded. It was created for the node epoch@localhost, and current node is aeternity@localhost (these must not differ!). To fix that, please go to your node's directory and run "bin/aeternity rename_db data"`. Running script and starting node worked.

**Comment by lucafavatella** [comment]
By setting in config key chain > db_path to an absolute path I got `Database cannot be loaded. It was created for the node epoch@localhost, and current node is aeternity@localhost (these must not differ!). To fix that, please go to your node's directory and run "bin/aeternity rename_db /Users/<snip>/test-db-renaming/2/testcfg"`. Renaming DB and starting node (reading top) worked.

**Comment by lucafavatella** [comment]
System tests being run in [this auxiliary CI job](https://circleci.com/gh/aeternity/aeternity/13762).

File: apps/aecore/src/aec_db.erl [hunk 1]
```diff
@@ -761,10 +761,8 @@ assert_schema_node_name(#{persist := true}) ->
     case DbOwnerNode =:= node() of
         true -> ok;
         false ->
-            DbDir = case aeu_env:user_config([<<"chain">>, <<"db_path">>]) of
-                        undefined -> "data";
-                        {ok, Path} -> Path
-                    end,
+            {ok, DbDir} = aeu_env:find_config([<<"chain">>, <<"db_path">>],
+                                              [user_config, schema_default]),
```
Comment (zp-sd) [inline apps/aecore/src/aec_db.erl]: Acutually what is used in the code is https://github.com/aeternity/aeternity/blob/master/apps/aecore/src/aecore_app.erl#L62, it does take into account `schema_default`.

**Pull request merged**